src/date-picker/embedded.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { DatePickerProps } from './interfaces';
+import Calendar, { DayIndex } from './calendar';
+import { memoizedDate } from './calendar/utils/date';
+import { CalendarTypes } from './calendar/definitions';
+
+export const DatePickerEmbedded = ({
+  calendarRef,
+  selectedDate,
+  focusedDate,
+  displayedDate,
+  normalizedLocale,
+  normalizedStartOfWeek,
+  isDateEnabled,
+  calendarHasFocus,
+  nextMonthAriaLabel,
+  previousMonthAriaLabel,
+  todayAriaLabel,
+  onChangeMonthHandler,
+  onSelectDateHandler,
+  onDateFocusHandler,
+}: DatePickerEmbeddedProps) => {
+  return (
+    <Calendar
+      ref={calendarRef}
+      selectedDate={memoizedDate('value', selectedDate)}
+      focusedDate={memoizedDate('focused', focusedDate)}
+      displayedDate={memoizedDate('displayed', displayedDate)}
+      locale={normalizedLocale}
+      startOfWeek={normalizedStartOfWeek}
+      isDateEnabled={isDateEnabled ? isDateEnabled : () => true}
+      calendarHasFocus={calendarHasFocus}
+      nextMonthLabel={nextMonthAriaLabel}
+      previousMonthLabel={previousMonthAriaLabel}
+      todayAriaLabel={todayAriaLabel}
+      onChangeMonth={onChangeMonthHandler}
+      onSelectDate={onSelectDateHandler}
+      onFocusDate={onDateFocusHandler}
+    />
+  );
+};
+
+export interface DatePickerEmbeddedProps {
```

Comment:
I think it does make sense to derive this props from the DatePicker like:

```
export type DatePickerEmbeddedProps = Exclude<DatePickerProps, 'name' | 'ariaLabel' | ...>
```

This would make it easier to update and to see what properties are unique, like calenderRef, which we'd need to turn to some internal one if decided to share the embedded component as a standalone.